src/main/java/io/curity/identityserver/plugins/action/lookupaccount/LookupAccountAuthenticationAction.java

```diff
@@ -67,6 +73,9 @@ public AuthenticationActionResult apply(AuthenticationActionContext context) {
         } else if (accountAttributes == null) {
             // Continue with the authentication flow even if the account is not found.
             return AuthenticationActionResult.successfulResult(authenticationAttributes);
+        } else if (_includeDevices){
+            // Add devices
+            accountAttributes.withDevices(getDevices(accountAttributes.getUserName()).toArray(Device[]::new));
```

Comment:
I think there may be use cases where there are devices attached to a username, but without having an account. So perhaps we should do this lookup even if no account was found?

Reply:
That would require a server change as well, currently the username device lookup is based on account attributes, do you think that would be important to have?